apps/zerg/ts-agent/.env.example

```
@@ -0,0 +1,10 @@
+# OpenAI API Key (for supervisor and development agent)
+OPENAI_API_KEY=your_openai_api_key_here
+
+# Google Gemini API Key (for knowledge base agent)
+GOOGLE_API_KEY=your_google_gemini_api_key_here
+
+# Optional: LangSmith for tracing (if you want to use it)
+LANGCHAIN_TRACING_V2=true
+LANGCHAIN_API_KEY=your_langsmith_api_key_here
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The LANGCHAIN_API_KEY key should go before the LANGCHAIN_TRACING_V2 key

(UnorderedKey)